staging/src/k8s.io/client-go/testing/actions.go

```diff
@@ -319,6 +319,18 @@ type DeleteAction interface {
 	GetName() string
 }
 
+type DeleteCollectionAction interface {
```

Comment:
No, more for consistency.
That generic iface and PatchAction were missing somehow...